src/proto/flwr/proto/driver.proto

```diff
@@ -56,3 +65,18 @@ message PullTaskResRequest {
   repeated string task_ids = 2;
 }
 message PullTaskResResponse { repeated TaskRes task_res_list = 1; }
+
+// PullServerAppIns messages
+message PullServerAppInsRequest { uint64 run_id = 1; }
+message PullServerAppInsResponse {
```

Comment:
worth keeping the same order as in https://github.com/adap/flower/blob/4970e429b35a9f1d1300427ed399330c6d08f625/src/proto/flwr/proto/clientappio.proto#L52? (excluding `Message`).

Reply:
I agree, let's keep the order for symmetry:
```python
message PullServerAppInsResponse {
  Context context = 1;
  Run run = 2;
  Fab fab = 3;
}
```